app-frontend/config/webpack/help_docs.json

```diff
@@ -0,0 +1,98 @@
+{
```

Comment:
this file should not be tracked - for testing purposes, you can link / embed it in the PR, but we should treat this the same as the overrides file, with an example template, not linking to real stuff